i18n/locale-en.json

```diff
@@ -209,15 +214,52 @@
   "I18N_PLAYER_FORWARD": "Forward",
   "I18N_PLAYER_BACK": "Back",
   "I18N_PLAYER_CONTINUE_NEXT_CARD": "Continue to next card",
+  "I18N_ERROR_DISABLED_EXPLORATION_PAGE_TITLE": "Disabled Exploration - Oppia",
   "I18N_ERROR_DISABLED_EXPLORATION": "Disabled Exploration",
   "I18N_ERROR_DISABLED_EXPLORATION_MESSAGE": "Sorry, but the exploration you clicked on is currently disabled. Please try again later.",
-  "I18N_ERROR_RECOMMENDATION": "The best thing to do now is probably to return to the <a href=\"/\">home page</a>. However, if this issue recurs, and you think it shouldn't, please let us know on our <a href=\"https://github.com/oppia/oppia/issues/new?title=Describe%20your%20feature%20request%20or%20bug%20report%20succinctly&body=If%20you%27d%20like%20to%20propose%20a%20feature,%20describe%20what%20you%27d%20like%20to%20see.%20Mock%20ups%20would%20be%20great!%0A%0AIf%20you%27re%20reporting%20a%20bug,%20please%20be%20sure%20to%20include%20the%20expected%20behaviour,%20the%20observed%20behaviour,%20and%20steps%20to%20reproduce%20the%20problem.%20Console%20copy-pastes%20and%20any%20background%20on%20the%20environment%20would%20also%20be%20helpful.%0A%0AThanks!\" target=\"_blank\">issue tracker</a>. Sorry about this.",
+  "I18N_ERROR_NEXT_STEPS": "The best thing to do now is probably to return to the <a href=\"/\">home page</a>. However, if this issue recurs, and you think it shouldn't, please let us know on our <a href=\"https://github.com/oppia/oppia/issues/new?title=Describe%20your%20feature%20request%20or%20bug%20report%20succinctly&body=If%20you%27d%20like%20to%20propose%20a%20feature,%20describe%20what%20you%27d%20like%20to%20see.%20Mock%20ups%20would%20be%20great!%0A%0AIf%20you%27re%20reporting%20a%20bug,%20please%20be%20sure%20to%20include%20the%20expected%20behaviour,%20the%20observed%20behaviour,%20and%20steps%20to%20reproduce%20the%20problem.%20Console%20copy-pastes%20and%20any%20background%20on%20the%20environment%20would%20also%20be%20helpful.%0A%0AThanks!\" target=\"_blank\">issue tracker</a>. Sorry about this.",
```

Comment:
FYI, we can now use the simple link

    https://github.com/oppia/oppia/issues/new

since GitHub has recently introduced default templates for issues.

Even better, though, would be to have the link passed in as a parameter to the I18N string, since it really shouldn't be translated. Up to you, though.